src/types/leafygreen.ts

```diff
@@ -1,26 +0,0 @@
-import Card from "@leafygreen-ui/card";
-import { H1, H2, H3, Overline, Subtitle } from "@leafygreen-ui/typography";
```

Comment:
(placement of comment is random) Does `SearchBarGuideCue/index.tsx` need `InlineKeyCodeProps`?